### PR TITLE
Update yum repo instructions in README

### PR DESCRIPTION
**Pull request by alexblackie**
There were a few issues with the yum repo configuration examples and instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,13 +99,14 @@ For rpm-based systems RH, CentOS, Fedora, add this in `/etc/yum.repos.d/logentri
 	name=Logentries repo
 	enabled=1
 	metadata_expire=1d
-	baseurl=http://rep.logentries.com/XXX/\$basearch
-	gpgkey=http://rep.logentries.com/RPM-GPG-KEY-logentries
+	baseurl=https://rep.logentries.com/XXX/\$basearch
+	gpgkey=https://rep.logentries.com/RPM-GPG-KEY-logentries
 
-Replace `XXX` with the name of your system, i.e. one of fedora18, fedora19,
```
Comment (ilyabiryukov-r7) [inline README.md]: Why remove the mention of `fedora18-20`?

Comment (ilyabiryukov-r7) [reply README.md]: The explanation is hidden in the commits, I see :)

**Comment by ilyabiryukov-r7** [comment]
I'd be nice to mention that you can go to `https://rep.logentries.com` to browse the full list of supported OS's

**Comment by ilyabiryukov-r7** [comment]
@alexblackie thanks for the contribution!

**Comment by alexblackie** [comment]
@ilyabiryukov-r7 Thanks for the quick response and feedback! :slightly_smiling_face:

**Pull request merged**